pkg/scheduler/predicates/ha_test.go

```diff
@@ -1182,6 +1182,15 @@ func getTopologies(nodes []apiv1.Node, topologyKey string) []string {
 	return arr
 }
 
+func getSortedTopologies(nodes []apiv1.Node, topologyKey string) []string {
+	arr := make([]string, 0)
+	for _, node := range nodes {
+		arr = append(arr, node.Labels[topologyKey])
+	}
```

Comment:
reuse `getTopologies` or remove `getTopologies`?

Reply:
remove `getTopologies `, replace `getTopologies ` with `getSortedTopologies`？

Reply:
I think either is fine